examples/fargate-serverless/main.tf

```diff
@@ -77,6 +78,12 @@ module "eks" {
     }
   }
 
+  fargate_profile_defaults = {
+    iam_role_additional_policies = {
+      additional = module.eks_blueprints_addons.fargate_fluentbit_policy[0].arn
```

Comment:
create the policy in the example

Reply:
This is the policy that allows the logStream creation. Without that the `fluentbit` add-on doesn't work properly.